index.html

```diff
@@ -107,154 +114,161 @@ <h2 class="work_primary_text">Multi-Post Stories</h2>
                     </div>
                 </div>
             </div>
-            <div class="grid-item item2">
-                <div class="content">
-                    <h3 class="p-text">Professional Art <br>
-                        Printing Data
-                    </h3>
-                    <p class="s-text">
-                        A daily selection of privately
-                        personalized reads; no accounts or
-                        sign-ups required. Has been the 
-                        industry's standard.
-                    </p>
-                    <div class="tags">
-                        <ul class="tag">
-                            <li> <a href="#">HTML</a></li>
-                            <li> <a href="#">Bootstrap</a></li>
-                            <li> <a href="#">Ruby</a></li>
-                        </ul>
-                    </div>
 
-                    <div class="btn2-holder">
-                        See Project
+            <section class="grid-desktop">
+
+                <div class="grid-item item2">
```

Comment:
- [ ] Please 🙏🏻  use the `<ul>` tag over `div` to enhance the professionalism and semantic correctness of your code, use the `<ul>` tag as a container and place each social media icon link within an individual **li** tag. This approach not only improves the structure of your *HTML* but also benefits **visually-impaired** individuals, ensuring a better user experience. Avoid using the `<div>` tag solely for decorative purposes, as it lacks semantic meaning.

   - Once you make the necessary changes, remember to add appropriate styles to your CSS file. Implementing 
     **flexbox** can help you properly align the items within the **ul** container. Utilizing `Flexbox` provides a flexible 
     and responsive layout, ensuring consistent presentation across different screen sizes and devices.

   - It is crucial to always close your tags with the correct tag name. This practice becomes even more significant as your code scales or when you add more features in the future. By diligently closing your tags, you avoid 
     potential errors that may arise due to small oversights. Maintaining proper tag closure ensures code readability, makes debugging easier, and helps maintain the integrity of your markup.